src/include/daos_srv/vos_types.h

```diff
@@ -207,27 +207,35 @@ typedef enum {
 
 enum {
 	/** Conditional Op: Punch key if it exists, fail otherwise */
-	VOS_OF_COND_PUNCH	= DAOS_COND_PUNCH,
+	VOS_OF_COND_PUNCH		= DAOS_COND_PUNCH,
 	/** Conditional Op: Insert dkey if it doesn't exist, fail otherwise */
-	VOS_OF_COND_DKEY_INSERT	= DAOS_COND_DKEY_INSERT,
+	VOS_OF_COND_DKEY_INSERT		= DAOS_COND_DKEY_INSERT,
 	/** Conditional Op: Update dkey if it exists, fail otherwise */
-	VOS_OF_COND_DKEY_UPDATE	= DAOS_COND_DKEY_UPDATE,
+	VOS_OF_COND_DKEY_UPDATE		= DAOS_COND_DKEY_UPDATE,
 	/** Conditional Op: Fetch dkey if it exists, fail otherwise */
-	VOS_OF_COND_DKEY_FETCH	= DAOS_COND_DKEY_FETCH,
+	VOS_OF_COND_DKEY_FETCH		= DAOS_COND_DKEY_FETCH,
 	/** Conditional Op: Insert akey if it doesn't exist, fail otherwise */
-	VOS_OF_COND_AKEY_INSERT	= DAOS_COND_AKEY_INSERT,
+	VOS_OF_COND_AKEY_INSERT		= DAOS_COND_AKEY_INSERT,
 	/** Conditional Op: Update akey if it exists, fail otherwise */
-	VOS_OF_COND_AKEY_UPDATE	= DAOS_COND_AKEY_UPDATE,
+	VOS_OF_COND_AKEY_UPDATE		= DAOS_COND_AKEY_UPDATE,
 	/** Conditional Op: Fetch akey if it exists, fail otherwise */
-	VOS_OF_COND_AKEY_FETCH	= DAOS_COND_AKEY_FETCH,
+	VOS_OF_COND_AKEY_FETCH		= DAOS_COND_AKEY_FETCH,
 	/** replay punch (underwrite) */
-	VOS_OF_REPLAY_PC	= (1 << 7),
+	VOS_OF_REPLAY_PC		= (1 << 7),
 	/* critical update - skip checks on SCM system/held space */
-	VOS_OF_CRIT		= (1 << 8),
+	VOS_OF_CRIT			= (1 << 8),
 	/** Instead of update or punch of extents, remove all extents
 	 * under the specified range. Intended for internal use only.
 	 */
-	VOS_OF_REMOVE		= (1 << 9),
+	VOS_OF_REMOVE			= (1 << 9),
+	/* only query iod_size */
+	VOS_FETCH_SIZE_ONLY		= (1 << 10),
```

Comment:
ah, missed this.   Should be VOS_OF (op flags?)  rather than just VOS for consistency.

Reply:
okay, it makes sense. I will fix this